### DISCUSSION TITLE
Pen Holder For Satsana Fan Duct

### DISCUSSION THREAD

**Discussion by adsaxton** [Sakati84/3DPrintColorizer#61] [Show and tell]
I have added a Remix to Thingaverse for the Pen Holder( https://www.thingiverse.com/thing:4943472). The mount for your 3DTouch holder is almost as think as the Satsana 3DTouch mount so I combined them! This works perfectly! The long version was necessary to clean the fan duct.

The offsets for this mount are X=42 & Y=54. Z will need to be measured.

**Comment by Sakati84** [comment]
Very nice! Can I put this in the GitHub?

**Comment by adsaxton** [comment]
Absolutely!
